spec/rails_helper.rb

```diff
@@ -5,6 +5,9 @@
 require 'rspec/rails'
 require 'capybara/rspec'
 require 'capybara/email/rspec'
+require 'sidekiq/testing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.